src/perf/dbserver/QuicPerformanceDataServer/Data/DbInitializer.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.EntityFrameworkCore.Internal;
+using Newtonsoft.Json;
+using QuicDataServer.Models;
+using QuicDataServer.Models.Db;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace QuicDataServer.Data
+{
+    public static class DbInitializer
+    {
+        public static async Task Initialize(PerformanceContext context)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            await context.Database.EnsureCreatedAsync().ConfigureAwait(false);
+            if (context.Platforms.Any())
+            {
+                return; // DB has been seeded
+            }
+
+            context.Platforms.Add(new DbPlatform
+            {
+                PlatformName = "Windows_x64_stub",
+            });
+
+            context.Platforms.Add(new DbPlatform
+            {
+                PlatformName = "Linux_x64_openssl"
+            });
+
+            await context.SaveChangesAsync().ConfigureAwait(false);
+            
+
+            var plat = context.Platforms.Where(x => x.PlatformName == "Windows_x64_stub").FirstOrDefault();
+
+            plat.Tests = new List<DbTest>();
+
+            plat.Tests.Add(new DbTest
+            {
+                TestName = "loopback",
+            });
+
+            plat = context.Platforms.Where(x => x.PlatformName == "Linux_x64_openssl").FirstOrDefault();
+
+            plat.Tests = new List<DbTest>();
+
+            plat.Tests.Add(new DbTest
+            {
+                DbPlatformId = 2,
+                TestName = "loopback",
+            });
+
+            await context.SaveChangesAsync().ConfigureAwait(false);
+
+            using HttpClient client = new HttpClient();
+
+            var seedUri = new Uri("https://raw.githubusercontent.com/ThadHouse/msquic/dbseed/seeddata.json");
```

Comment:
Should this line/location be in here? Also, do we really need this initializer code checked in at all?

Reply:
Removed. I can always recreate it if necessary.